pkgs/development/compilers/cudatoolkit/redist/build-cuda-redist-package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+}:
+
+pname:
+attrs:
+
+let
+  arch = "linux-x86_64";
+in stdenv.mkDerivation {
+  inherit pname;
+  inherit (attrs) version;
+
+  src = assert (lib.hasAttr arch attrs); fetchurl {
+    url = "https://developer.download.nvidia.com/compute/cuda/redist/${attrs.${arch}.relative_path}";
+    inherit (attrs.${arch}) sha256;
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+  ];
+
+  dontBuild = true;
+
+  # TODO: choose whether to install static/dynamic libs
+  installPhase = ''
+    runHook preInstalHook
+    rm LICENSE
+    mkdir -p $out
+    mv * $out
+    runHook postInstallHook
+  '';
+
+  meta = {
```

Comment:
Is there a way to specify a team like @NixOS/cuda-maintainers as a meta.maintainer for a package?

Reply:
we can add a team in maintainers-list.nix

Reply:
this isn't directly related to these changes, so we can always do this in a separate PR